modules/web/src/de/diedavids/cuba/dataimport/web/importwizard/ImportWizard.groovy

```diff
@@ -144,32 +178,35 @@ class ImportWizard extends AbstractWindow {
     }
 
     private ImportConfiguration createImportConfiguration() {
-        def importConfiguration = metadata.create(ImportConfiguration)
-        importConfiguration.dateFormat = 'dd/MM/yyyy'
-        importConfiguration.booleanTrueValue = 'Yes'
-        importConfiguration.booleanFalseValue = 'No'
-        importConfiguration
+        defaultImportConfiguration = metadata.create(ImportConfiguration)
```

Comment:
e.g. this part can go into @PostConstruct methods of ImportConfiguration itself --> reduces the size of the wizard